zerver/lib/request.py

```diff
@@ -155,7 +155,7 @@ def _wrapped_view_func(request, *args, **kwargs):
                 try:
                     val = ujson.loads(val)
                 except Exception:
-                    raise JsonableError(_('argument "%s" is not valid json.') % (param.post_var_name,))
+                    raise JsonableError(_('Argument "%s" is not valid JSON.') % (param.post_var_name,))
```

Comment:
I completetly forgot to run the backend tests after these changes. Thanks for the heads up!